pkg/bindings/containers/containers.go

```diff
@@ -374,11 +378,55 @@ func Attach(ctx context.Context, nameOrId string, detachKeys *string, logs, stre
 		params.Add("stderr", "true")
 	}
 
+	if terminal.IsTerminal(int(os.Stdin.Fd())) && ctnr.Config.Tty {
```

Comment:
Should this be `os.Stdin` or the stdin we passed in?

Reply:
fixed, using stdin parameter.